spec/features/create_ride_spec.rb

```diff
@@ -0,0 +1,20 @@
+require "rails_helper"
+require "active_support"
+
+feature 'User creates a ride' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.